Capes/loadCape/toggleEach.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
If overlays are required to be loaded or hardware is to be installed, please add comments to that effect.

Reply:
Oh, and send the pull request to v2020.08, not v2020.01!